### PR TITLE
Add a dockerfile for elasticsearch 0.90

### PR DESCRIPTION
**Pull request by gpocentek**

### CODE REVIEW

File: docker/images/es-0.90/config/logging.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+es.logger.level: INFO
+rootLogger: ${es.logger.level}, console
+logger:
+  # log action execution errors for easier debugging
+  action: DEBUG
+  # reduce the logging for aws, too much is logged under the default INFO
+  com.amazonaws: WARN
```
Comment (jmaupetit) [inline docker/images/es-0.90/config/logging.yml]: Are we concerned by this particular logger?

File: docker/images/es-0.90/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+     apt-get update && \
+     apt-get install -y --no-install-recommends elasticsearch=$ELASTICSEARCH_VERSION && \
+     rm -rf /var/lib/apt/lists/*
+
+ENV PATH /usr/share/elasticsearch/bin:$PATH
+
+WORKDIR /usr/share/elasticsearch
+
+COPY config ./config
+
```
Comment (jmaupetit) [inline docker/images/es-0.90/Dockerfile]: Can you add a comment here? Something like: "Create ES tree and set permissions"

File: docker/images/es-0.90/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        ./data \
+        ./logs \
+        ./config \
+        ./config/scripts; do \
+            mkdir -p "$path"; \
+            chgrp -R 0 "$path"; \
+            chmod -R g=u "$path"; \
+        done
+
+USER elasticsearch
```
Comment (jmaupetit) [inline docker/images/es-0.90/Dockerfile]: Is this user created when installing the ad hoc package? If yes, can you add a comment about this user creation and permissions/groups he belongs to?

File: docker/images/es-0.90/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+COPY config ./config
+
+RUN for path in \
+        ./data \
+        ./logs \
+        ./config \
+        ./config/scripts; do \
+            mkdir -p "$path"; \
+            chgrp -R 0 "$path"; \
+            chmod -R g=u "$path"; \
```
Comment (jmaupetit) [inline docker/images/es-0.90/Dockerfile:31]: So the `elasticsearch` user belongs to the `root` group?

File: docker/images/es-0.90/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,33 @@
+FROM java:8-jre
+
+RUN apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-keys 46095ACC8548582C1A2699A9D27D666CD88E42B4
+
+ENV ELASTICSEARCH_MAJOR 0.90
+ENV ELASTICSEARCH_VERSION 0.90.13
+ENV ELASTICSEARCH_REPO_BASE http://packages.elasticsearch.org/elasticsearch/0.90/debian
```
Comment (jmaupetit) [inline docker/images/es-0.90/Dockerfile]: DRY?
```Dockerfile
ENV ELASTICSEARCH_VERSION ${ELASTICSEARCH_MAJOR}.13 
ENV ELASTICSEARCH_REPO_BASE http://packages.elasticsearch.org/elasticsearch/${ELASTICSEARCH_MAJOR}/debian
```

**Review by jmaupetit — COMMENTED** [review]
Thank you for this great work! Almost good to go. I think adding a few comments will help to better understand what you are trying to achieve.

**Comment by jmaupetit** [comment]
Also, don't forget to update the documentation (README) for this new service.

**Review by sampaccoud — APPROVED** [review]

**Review by jmaupetit — APPROVED** [review]

**Comment by jmaupetit** [comment]
Same thing here, you'll need to force push this branch to trigger the CI build.

**Pull request merged**